spec/tasks/gen/resource_spec.cr

```diff
@@ -4,7 +4,7 @@ include CleanupHelper
 include GeneratorHelper
 
 describe Gen::Action do
-  it "generates actions, model, form and query" do
+  it "generates actions, model, operation and query" do
```

Comment:
Might as well change this while I'm in here 😂

Reply:
Nice!